relay-server/src/actors/processor.rs

```diff
@@ -1871,6 +1873,9 @@ impl EnvelopeProcessorService {
             );
         }
 
+        let mut android_csv = PathBuf::from(self.config.path());
```

Comment:
This won't compile the file into Relay, which we decided to do if I recall it right.  This will always try to open and read the file from the path. 

You have to look into something like [`include_str!`](https://doc.rust-lang.org/std/macro.include_str.html) or `include_bytes!` macros.